app/models/mentor_request.rb

```diff
@@ -0,0 +1,32 @@
+class MentorRequest < ActiveRecord::Base
+  belongs_to :student, class_name: "User"
+  validates_presence_of :student
+
+  after_create :light_up
+
+  def light_up
+    Light.on
+  end
+    
+
+  def self.open_by_user(user)
+    req = self.where(student: user, open: true)
+  end
+
+  def self.close_all_by_user(user)
+    self.open_by_user(user).each{|req| req.close}
+  end
```

Comment:
It seems like it might be better to do this with:

`````ruby
self.open_by_user(user).update_all(open: false)
`````
and then handle the other side-effects of #close (conditional Light.off)  manually.